.github/workflows/ci.yml

```diff
@@ -11,14 +11,15 @@ jobs:
         - '1.18.x'
         - '1.19.x'
         - '1.20.x'
+        - '1.21.x'
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
```

Comment:
confirmed this is the latest major vers of github `actions/checkout` https://github.com/actions/setup-go